packages/users/src/api/index.ts

```diff
@@ -17,6 +17,8 @@ import {
   ReadInboxMessageResponse,
   DeleteInboxMessageRequest,
   DeleteSentboxMessageRequest,
+  GetUsageRequest,
```

Comment:
IMO, I would import these an some internal name, then use the nice names as the response type for the user.

Reply:
We don't make the users use those, https://github.com/textileio/js-textile/pull/722/files#diff-0c6964ee0de0d6937c3fc3d2250393adf892a2893bb921b776dfcded5f650d8aR390 though perhaps a nicer name that GetUsageResponsObj is still in order

Reply:
Right, I was just suggesting do something like `GetUsageRequest as GetUsageRequestInternal` or whatever the JS way of doing that is now days, then use the pleasant name `GetUsageRequest` as your custom type (the return type)

Reply:
oh nice, good call

Reply:
An alternative here is to import it like this:

```
import * as pb from "@textile/users-grpc/api/usersd/pb/usersd_pb"
```

And then prefix everything with `pb.` when using them. this is how we do it in threads IIRC? Not saying that's the right way, just an option. The way @andrewxhill has it here might be better for tree-shaking?

Reply:
(though of course, import "as" internal name is totally a good call)

Reply:
"import * as" was causing issues, i think it was with api-extractor so have avoided it. updated with `Thing as _Thing` for now. Updated other cases of `ThingObj` and added backward compatibility where I saw them.